spring-cloud-gcp-docs/src/main/asciidoc/config.adoc

```diff
@@ -0,0 +1,101 @@
+== Spring Cloud Config
+
+Spring Cloud GCP makes it possible to use the
+https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/[Google
+Runtime Configuration API] as a
+https://cloud.spring.io/spring-cloud-config/[Spring Cloud Config] server to remotely store your
+application configuration data.
+
+The Spring Cloud GCP Config support is provided via a Spring Boot starter that makes notation like
```

Comment:
Config support => Config client support

Reply:
I omitted the "client" bit on purpose because I don't want readers to think they can use the generic Spring Cloud Config client to connect to the Google Runtime Config API. The Spring Cloud Config client actually can't be used to get config information from us (which is why it was a good idea in the end to not re-use the Spring Cloud Config property namespace `spring.cloud.config.*`).

Reply:
I think in that case, it might best to rephrase the server part as "Configuration server" and not imply that it may be Git backed.

Reply:
Because we don't actually use Spring Cloud Config for anything (i.e., we basically re-implement it), I wonder if it's worth rewording to say something like "storing configuration data in a similar way to Spring Cloud Config"?
I'm also not comfortable with the Spring Cloud Config server bit.